python/tests/test_cog.py

```diff
@@ -1,27 +1,31 @@
-import async_tiff
 from time import time
-from async_tiff import TIFF
+
+from async_tiff import TIFF, enums
 from async_tiff.store import S3Store
 
-store = S3Store("sentinel-cogs", region="us-west-2", skip_signature=True)
-path = "sentinel-s2-l2a-cogs/12/S/UF/2022/6/S2B_12SUF_20220609_0_L2A/B04.tif"
 
-tiff = await TIFF.open(path, store=store, prefetch=32768)
+async def test_cog_s3():
+    """
+    Ensure that TIFF.open can open a Sentinel-2 Cloud-Optimized GeoTIFF file from an
+    s3 bucket, read IFDs and GeoKeyDirectory metadata.
+    """
+    store = S3Store("sentinel-cogs", region="us-west-2", skip_signature=True)
+    path = "sentinel-s2-l2a-cogs/12/S/UF/2022/6/S2B_12SUF_20220609_0_L2A/B04.tif"
+
+    start = time()
+    tiff = await TIFF.open(path=path, store=store, prefetch=32768)
+    end = time()
+    assert end - start < 5  # take less than 5 seconds to open
```

Comment:
I don't think we need to add this assert into the actual test. We can just take out the timing

Reply:
Done in commit 28973b91a693c7b2fb6bfc9ba1d82020f5927e03